Sources/URLSessionAPIAdapter.swift

```diff
@@ -8,6 +8,51 @@
 
 import Foundation
 
+// This class provides user with easy way to serialize access to a property in multiplatform environment. This class is written with future PropertyWrapper feature of swift in mind.
+internal final class Serialized<Value> {
```

Comment:
a. I would separate this class to a file. It is a general use thing.
b. Internal is the default access control so there is no need to specify it.
```suggestion
final class Serialized<Value> {
```